change/@fluentui-eslint-plugin-e6e0684b-1ee7-4695-a151-9f716d78ab67.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "enable autofixable import rules",
+  "packageName": "@fluentui/eslint-plugin",
+  "email": "martinhochel@microsoft.com",
+  "dependentChangeType": "patch"
```

This should be none to avoid bumping all packages
```suggestion
  "dependentChangeType": "none"
```